app/src/NativeBase/Components/ColouredTag.tsx

```diff
@@ -0,0 +1,39 @@
+/**
+ * @file shows text in a coloured box, randomly choosing one of four colours
+ */
+
+import React from 'react'
+import { Box } from 'native-base'
+
+type ColouredTagProps = {
+  title: string
+}
+
+/**
+ * A functional component to display the project role title in a coloured tag
+ * @param {ColouredTagProps} props - the props for this component
+ * @param {string} props.role - the role title
```

Comment:
Tiny thing but should this description be "the text" rather than "the role title"?